src/test/java/io/seqera/tower/cli/shared/WorkflowDumpExportFormatTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2021-2023, Seqera.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.seqera.tower.cli.shared;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.seqera.tower.model.Workflow;
+import org.junit.jupiter.api.Test;
+
+import static io.seqera.tower.cli.utils.JsonHelper.parseJson;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class WorkflowDumpExportFormatTest {
+
+    @Test
+    void testFormatDeserialization() throws JsonProcessingException {
+        String json = "{\n" +
```

Comment:
I think that this pattern of hardcoding huge chunks of text (in this case even worse since it is Json and we need to escape the ") should be avoided. We could add the json files to the resources folder (maybe using the test name as a folder name, to separate the resources from other tests) and load them. That way I think the class becomes more readable and less error prone. What do you think?